packages/nx/src/utils/output.ts

```diff
@@ -29,18 +30,9 @@ export interface CLISuccessMessageConfig {
   bodyLines?: string[];
 }
 
-/**
- * Automatically disable styling applied by chalk if CI=true
- */
-const forceColor =
-  process.env.FORCE_COLOR === '' || process.env.FORCE_COLOR === 'true';
-if (isCI() && !forceColor) {
```

Comment:
picocolors checks for NO_COLOR. otherwise there's no way to force the colours off as far as I know

that feels like the right thing to do, though.

maybe we should just hope/assume CI sets NO_COLOR? or document that people should if they want to